extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -110,11 +118,17 @@ public function match( string $type, $content ) {
 	 *
 	 * @param array $changed -- list of rules to reload, default empty array so reload all
 	 *
+	 * @return int|mixed
 	 */
-	public function reload( $changed = array() ) {
-		wfProfileIn( __METHOD__  );
-		$result = $this->sendToPhalanxDaemon( "reload", is_array( $changed ) && sizeof( $changed ) ? array( "changed" => implode( ",", $changed ) ) : array() );
-		wfProfileOut( __METHOD__  );
+	public function reload( $changed = [] ) {
+		wfProfileIn( __METHOD__ );
+
+		$params = is_array( $changed ) && sizeof( $changed )
+			? [ "changed" => implode( ",", $changed ) ]
+			: [];
+
+		$result = $this->sendToPhalanxDaemon( "reload", $params );
+		wfProfileOut( __METHOD__ );
```

Comment:
I updated the above to set a `$params` variable to bring the line length under the 120 char limit.